angular-spa-ui/js/search-app/search-module/search.jade

```diff
@@ -1 +1 @@
-p {{greeting}}
\ No newline at end of file
+<h2>{{greeting}}</h2>
```

Comment:
Variable in brackets should be separated with space, e.g. <h2>{{ greeting }}</h2>